src/zarate.jl

```diff
@@ -21,7 +21,7 @@ In 2018 IEEE Pacific Visualization Symposium (PacificVis) (pp. 135-139). IEEE.
 """
 Base.@kwdef struct Zarate <: AbstractLayout
     time_limit::Dates.Period = Dates.Second(0)
-    ordering_solver::Any = optimizer_with_attributes(Cbc.Optimizer, "randomSeed"=>1, "randomCbcSeed"=>1)
+    ordering_solver::Any = optimizer_with_attributes(HiGHS.Optimizer, "random_seed" => 1)
     arranging_solver::Any = ECOS.Optimizer
```

Comment:
Can we use HiGHs for this too?
That would be great so we can drop a dependency

Reply:
I tried, but HiGHS does not support SOCP...

```julia
medium_pert direct: Error During Test at /home/sjkelly/.julia/dev/LayeredLayouts/test/demos.jl:47
  Got exception outside of a @test
  Constraints of type MathOptInterface.VectorAffineFunction{Float64}-in-MathOptInterface.RotatedSecondOrderCone are not supported by the solver.
  
  If you expected the solver to support your problem, you may have an error in your formulation. Otherwise, consider using a different solver.
  
  The list of available solvers, along with the problem types they support, is available at https://jump.dev/JuMP.jl/stable/installation/#Supported-solvers.
  Stacktrace:
    [1] error(s::String)
      @ Base ./error.jl:35
    [2] _moi_add_constraint(model::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.Bridges.LazyBridgeOptimizer{HiGHS.Optimizer}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, f::MathOptInterface.VectorAffi
    ```